onyo/lib/onyo.py

```diff
@@ -94,45 +96,79 @@ def __init__(self,
         # caches
         self._asset_paths: Optional[list[Path]] = None
 
+    def set_config(self,
+                   name: str,
+                   value: str,
+                   location: str = 'onyo') -> None:
+        """Set the configuration option `name` to `value`.
+
+        Parameters
+        ----------
+        name: str
+          The name of the configuration option to set.
+        value: str
+          The value to set for the configuration option.
+        location: Path or str, optional
```

Comment:
Half true ;-)

Removed `Path` - thx!
But: It is optional, because it comes with a default in the declaration. Does not require an `Optional` type annotation in this case.